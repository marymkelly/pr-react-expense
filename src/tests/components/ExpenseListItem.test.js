import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';
import  ExpenseListItem  from '../../components/pages/ExpenseListItem';
import expenses from '../fixtures/expenses';

test('Should render ExpenseListItem with correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
}) 