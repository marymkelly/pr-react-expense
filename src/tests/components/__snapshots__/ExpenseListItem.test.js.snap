// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem with correctly 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h4>
      Gum
    </h4>
  </Link>
  <p>
    195
     - 
    0
  </p>
</div>
`;
